gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -0,0 +1,508 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2013, Open Source Robotics Foundation
```

Comment:
worth updating the year in all these files

Reply:
These files come originally from https://github.com/ros-simulation/gazebo_ros_pkgs I don't think we can updated the year

Reply:
Yeah we've been keeping the original license headers as we port code to ROS 2. If it's a complete refactor though, we've been using Apache 2 for new code.

Reply:
Ah thanks, I thought it was a new addition